src/client/dfuse/ops/ioctl.c

```diff
@@ -297,6 +297,33 @@ handle_dooh_ioctl(struct dfuse_obj_hdl *oh, size_t size, fuse_req_t req)
 	DFUSE_REPLY_ERR_RAW(oh, req, rc);
 }
 
+static void
+handle_cont_query_ioctl(fuse_req_t req, const void *in_buf, size_t in_bufsz)
+{
+	struct dfuse_info            *dfuse_info = fuse_req_userdata(req);
+	struct dfuse_mem_query        query      = {};
+	const struct dfuse_mem_query *in_query   = in_buf;
+	int                           rc;
+
+	if (in_bufsz != sizeof(query))
+		D_GOTO(err, rc = EIO);
```

Comment:
Should this print a DEBUG? If it doesn't fail here, I imagine it would be tricky to track down

Reply:
DFUSE_REPLY_ERR_RAW() will do that.